backend/supabase/user_contract_metrics.sql

```diff
@@ -1,44 +1,46 @@
 create table if not exists
-    user_contract_metrics (
-      id bigint generated always as identity primary key,
-      user_id text not null,
-      contract_id text not null,
-      data jsonb not null,
-      fs_updated_time timestamp,
-      has_yes_shares boolean,
-      has_no_shares boolean,
-      total_shares_yes numeric,
-      total_shares_no numeric,
-      profit numeric,
-      has_shares boolean,
-      answer_id text
+  user_contract_metrics (
+    id bigint generated always as identity primary key,
+    user_id text not null,
+    contract_id text not null,
+    data jsonb not null,
+    fs_updated_time timestamp,
+    has_yes_shares boolean,
+    has_no_shares boolean,
+    total_shares_yes numeric,
+    total_shares_no numeric,
+    profit numeric,
+    has_shares boolean,
+    answer_id text
```

Comment:
It's generally bad to have these whitespace changes in the diff, because it obscures what was actually changed.